pytorch_lightning/plugins/training_type/ddp2.py

```diff
@@ -34,26 +35,25 @@ def setup(self, model):
         self.task_idx = self.cluster_environment.local_rank()
         # the difference to DDP is that we don't call children processes here
 
-    def reduce(self, tensor, *args, **kwargs):
+    def reduce(self, collection: _METRIC_COLLECTION, *args, **kwargs) -> _METRIC_COLLECTION:
```

Comment:
if ignored use this...
```suggestion
    def reduce(self, collection: _METRIC_COLLECTION, *_, **__) -> _METRIC_COLLECTION:
```